src/SwipeableListItem.js

```diff
@@ -25,17 +25,21 @@ class SwipeableListItem extends PureComponent {
     this.contentLeft = null;
     this.contentRight = null;
     this.listElement = null;
+    this.requestedAnimationFrame = null;
     this.wrapper = null;
 
     this.startTime = null;
 
+    this.prevSwipeDistancePercent = 0;
```

Comment:
Consider not using an abbreviation and instead use of the full name to aid edition and searchability:

```suggestion
    this.previousSwipeDistancePercent = 0;
```